app/src/main/java/com/saotome/imed_app/ui/MainActivity.kt

```diff
@@ -29,10 +29,10 @@ class MainActivity : AppCompatActivity() {
         appBarConfiguration = AppBarConfiguration(navController.graph)
         setupActionBarWithNavController(navController, appBarConfiguration)
 
-        binding.fab.setOnClickListener { view ->
```

Comment:
Dont comment this code. If you are keeping the "fab" button, just remove the code inside the click listener and add a TODO comment . 

Ex:

// TODO: This click listener will be implemented in PR X

Reply:
Thanks for the tip. Done
670270c3977d061440ddc4f91f242f72151084ed